zoom_monitor.py

```diff
@@ -154,6 +188,14 @@ def count_hands(self):
         """
         Return the number of people in the participants list with raised hands
         """
+        # If someone starts recording the meeting, we'll get a pop-up modal
+        # warning us about that before we can count hands again.
+        self._acknowledge_recording()
+
+        # If someone else shares their screen, it closes the participants list.
+        # So, try reopening it every time we want to count hands.
+        self._open_participants_list()
```

Comment:
There's a bit of a race condition here: if someone starts recording after we execute line 193 but before we execute line 197, we could be in trouble. For now, I'm content assuming that's rare enough that it won't happen. :shrug: